options.html

```diff
@@ -136,6 +136,37 @@
 {
   box-shadow: 0px 0px 10px #e03030;
 }
+#userFilterLists {
```

Comment:
Coding style: "Opening braces always go on their own line."

You can find our coding style at https://adblockplus.org/en/coding-style. Please also apply these changes to other parts of the code that don't comply.